packages/cli/tests/images/build.js

```diff
@@ -73,9 +73,7 @@ exports.prerender.heads.route66 = `
 	<link rel="preload" href="\\/route-route66\\.chunk\\.\\w{5}\\.js" as="script">
 	<link rel="preload" href="\\/route-route66\\.chunk\\.\\w{5}\\.css" as="style">
 	<link rel="shortcut icon" href="\\/favicon\\.ico">
-	<link href=\\"/bundle.\\w{5}.css\\" rel=\\"preload\\" as=\\"style\\" onload=\\"this.rel='stylesheet'\\"><noscript>
-		<link rel=\\"stylesheet\\" href=\\"\\/bundle.\\w{5}.css\\"><\\/noscript>
-	<style>html{padding:0;}<\\/style>
+	<link href=\\"\\/bundle.\\w{5}.css\\" rel=\\"stylesheet\\">
```

Comment:
Where did the noscript tag go?
Isn't it needed with critters

Reply:
I guess not. The `noscript` tag is the reason why the tests were failing. Either ways, closed this PR in favor of #696 (which covers these changes as well)